how-to-guides/fastai-lr-finder.ipynb

```diff
@@ -0,0 +1,436 @@
+    "This how-to guide demonstrates how we can leverage the [`FastaiLRFinder`](https://pytorch.org/ignite/generated/ignite.handlers.lr_finder.FastaiLRFinder.html) handler to find an optimal learning rate to train our model on. We will compare the results produced with and without using the handler for better understanding.\n",
+    "\n",
+    "<!--more-->"
+   ],
+   "metadata": {
+    "id": "5w-QlZE9mvdY"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this example, we will be using a [ResNet18](https://pytorch.org/vision/stable/models.html#torchvision.models.resnet18) model on the [MNIST](https://pytorch.org/vision/stable/datasets.html#torchvision.datasets.MNIST) dataset. The base code is the same as used in the [Getting Started Guide](https://pytorch-ignite.ai/getting-started)."
+   ],
+   "metadata": {
+    "id": "3Pq6c9nuRDXE"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Basic Setup"
+   ],
+   "metadata": {
+    "id": "L_wmAdFgmvdx"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "source": [
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchvision.datasets import MNIST\n",
+    "from torchvision.models import resnet18\n",
+    "from torchvision.transforms import Compose, Normalize, ToTensor\n",
+    "\n",
+    "from ignite.engine import create_supervised_trainer, create_supervised_evaluator\n",
+    "from ignite.metrics import Accuracy, Loss\n",
+    "from ignite.handlers import FastaiLRFinder"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "lMphyBmmmvdw",
+    "pycharm": {
+     "is_executing": false
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "source": [
+    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
+    "\n",
+    "\n",
+    "class Net(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super(Net, self).__init__()\n",
+    "\n",
+    "        self.model = resnet18(num_classes=10)\n",
+    "        self.model.conv1 = nn.Conv2d(\n",
+    "            1, 64, kernel_size=(3, 3), stride=(2, 2), padding=(3, 3), bias=False\n",
+    "        )\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        return self.model(x)\n",
+    "\n",
+    "\n",
+    "model = Net().to(device)\n",
+    "\n",
+    "data_transform = Compose([ToTensor(), Normalize((0.1307,), (0.3081,))])\n",
+    "\n",
+    "train_loader = DataLoader(\n",
+    "    MNIST(download=True, root=\".\", transform=data_transform, train=True),\n",
+    "    batch_size=128,\n",
+    "    shuffle=True,\n",
+    ")\n",
+    "\n",
+    "test_loader = DataLoader(\n",
+    "    MNIST(download=True, root=\".\", transform=data_transform, train=False),\n",
+    "    batch_size=256,\n",
+    "    shuffle=False,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "model = Net().to(device)\n",
+    "optimizer = torch.optim.RMSprop(model.parameters(), lr=1e-06)\n",
```

Comment:
Because I couldn't show a good enough improvement, only about 0.5% increase in accuracy. Mainly because it's hard to mess up MNIST. So I set the fixed lr very low.